tests/plugins/lti_consumer/test_api.py

```diff
@@ -177,10 +284,10 @@ def test_lti_consumer_view_set_get_context_method_is_cached(self, mock_params):
         request.toolbar = CMSToolbar(request)
         view_set = LTIConsumerViewsSet()
 
-        with self.assertNumQueries(1):
+        with self.assertNumQueries(9):
```

Comment:
Seems the right value is still `1`

Reply:
:man_shrugging: not on my laptop... reverting to old code.